.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+## Description
+
+Replace the content in this section with:
+- The motivation and context for this change (if it is not immediately clear from the title)
+- If it fixes an open issue, specify the issue number (e.g., "fixes #XXXX")
+- A summary of the behavior expected from this change
+- A description of tests performed
+
+## Author Progress Checklist:
+
+- [ ] Open draft pull request
+    - [ ] Make title clearly understandable in a standalone change log context
+    - [ ] Assign yourself the issue
+    - [ ] Add at least one label
+- [ ] Make code changes (if you haven't already)
```

Comment:
We should add linking to the issue(s) in the Development field of the PR:

![Screen Shot 2022-04-01 at 10 41 44 AM](https://user-images.githubusercontent.com/1964570/161307129-89bb390b-1205-455c-acb2-9153041807f9.png)